src/_i18n/es.yml

```diff
@@ -141,9 +141,14 @@ publications-post:
 services:
   title: Software de calidad entregado efectivamente
   subtitle: Lideres reconocidos en continuous delivery y prácticas de calidad y pericia.
```

Comment:
Pericia is not that used in Spanish (other than people that have played D&D in the Zinco print of the 2nd edition)

Reply:
Habilidad?